### PR TITLE
Use correct path to extension.neon

### PR DESCRIPTION
**Pull request by dantleech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ When not using [`phpstan/extension-installer`](https://github.com/phpstan/extens
 
 ```diff
  includes:
-+	- vendor/jangregor/phpstan-prophecy/extension.neon
++	- vendor/jangregor/phpstan-prophecy/src/extension.neon
```
Comment (localheinz) [inline README.md:32]: @dantleech 

That is actually not the correct path - it's only a symbolic link (for legacy reasons).

**Comment by localheinz** [comment]
Thank you, @dantleech!

File: README.md (hunk 1 above)
Comment (dantleech) [reply README.md:32]: hmm - the old one is the symbolic link?

the previous configuration didn't work for me.

Comment (localheinz) [reply README.md:32]: @dantleech

Yes, it should: https://github.com/Jan0707/phpstan-prophecy/blob/0.7.0/src/extension.neon

Does it not?

Comment (localheinz) [reply README.md:32]: @dantleech

Ha, sorry - that change is only included on [`0.7.0`](https://github.com/Jan0707/phpstan-prophecy/releases/tag/0.7.0), see https://github.com/Jan0707/phpstan-prophecy/commit/5fc5e3c8ae693f7fdc78bb1674a417c2874890cd#diff-300c6013fb9bc1bc02b8fbd9fb9ac2b0.

**Pull request closed without merging**